jupyter_book/book_template/Makefile

```diff
@@ -29,6 +29,9 @@ clean:
 serve:
 	bundle exec guard
 
-build:
+site:
+	jupyter-book build ./ --overwrite
+
+build: site
```

Comment:
Can we swap these? So that "site" is the current build commands, and "build" is the current "site"command? To me it feels more intuitive that the `build:` command will call `jupyter-book build`. WDYT?